E/ExperimentalDesign/Deps.toml

```diff
@@ -0,0 +1,13 @@
+[0]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
I'll just note that you probably don't need Documenter as a dependency.

Reply:
I added it as a dependency because I started using doctests as part of unit testing, but I'm not sure it's the best idea. I'll try to come up with a better solution, maybe creating a separate project for Travis testing? I like the idea of doctests being useful for package users and also useful for package testing, tough.

Reply:
You don't need to depend on Documenter for doctests.

Reply:
You do, if you want to use them as part of unit testing:
https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#Doctesting-as-Part-of-Testing-1

Reply:
No, you can just have it as a test-dependency in that case.

Reply:
Ok, I'll look into it.